.github/workflows/ci.yml

```diff
@@ -257,8 +261,8 @@ jobs:
             doc/source/api/core/meshing/datamodel
             doc/source/api/core/solver/tui
             doc/source/api/core/solver/datamodel
-          key: API-Code-v${{ env.API_CODE_CACHE }}-${{ steps.version.outputs.PYFLUENT_VERSION }}-${{ matrix.image-tag }}-${{ hashFiles('codegen/**') }}-${{ github.sha }}
-          restore-keys: API-Code-v${{ env.API_CODE_CACHE }}-${{ steps.version.outputs.PYFLUENT_VERSION }}-${{ matrix.image-tag }}-${{ hashFiles('codegen/**') }}
+          key: API-Code-v${{ env.API_CODE_CACHE }}-${{ steps.version.outputs.PYFLUENT_VERSION }}-${{ matrix.image-tag }}-${{ hashFiles('codegen/**') }}
```

Comment:
This should improve the caching strategy. Right now the codegen is running too frequently - with every new PR.

Reply:
Yeah, was there a reason you put `github.sha` in the key before?

Reply:
@dnwillia-work I originally copied this from [pymapdl](https://github.com/pyansys/pymapdl/blob/main/.github/workflows/ci.yml#L182).